remoteworkitem/remoteworkitem.go

```diff
@@ -78,6 +106,7 @@ func NewJiraRemoteWorkItem(item RemoteWorkItem) (AttributeAccesor, error) {
 		return nil, err
 	}
 	// TODO for sbose: Flatten !
+	j = Flatten(j)
```

Comment:
With this line in place, I'm guessing that the `TODO` comment above can be removed?